### PR TITLE
reformating

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR is only reformatting the documents with the build in "Format Document" functionality that comes with visual studio code

**Review:**
- [ ] code approved by
- [ ] tests executed by
- [ ] "merge and deploy" approved by

### CODE REVIEW

File: scout/server/blueprints/variants/templates/variants/variant.html [hunk 1]
```diff
@@ -12,1337 +12,1373 @@
...
-    <div class="list-group-item">
-      <strong>Gene panels</strong>:
-      {{ variant.panels|join(', ') }}
-    </div>
-    <div class="list-group-item disabled"><strong>Amino acid changes</strong></div>
-    {% for gene in variant.genes %}
-        {% for transcript in gene.primary_transcripts %}
-          {% for refseq_id in transcript.refseq_identifiers %}
-              {% if transcript.protein_sequence_name%}
-                <div class="list-group-item">
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/variant.html]: Here for instance the code would be much more complicated to read.

**Comment by mayabrandi** [comment]
Oki. Its just that i'm used to look at htmls in this format and I thought that, since it is what my editor is suggesting for htmls, it is what is common standard. But its not important to me. Ill get used to you htmls for sure :) You can close this PR without merging.

**Comment by northwestwitch** [comment]
Sorry I clicked on close by mistake. I don't know much about standards but I like the way we code now because it's easier to read. And if we change a page then we should change all templates, to be consistent. I'd rather leave as it is if you don't mind.

**Comment by northwestwitch** [comment]
Closing this!

**Comment by mayabrandi** [comment]
Fine!

**Pull request closed without merging**